codex-ui/src/vue/card-vertical/CardVertical.vue

```diff
@@ -1,9 +1,13 @@
 <template>
-  <div :class="$style['note']">
-    <div :class="$style['cover']"></div>
-    <div :class="$style['footer']">
-      <Heading :level="3">{{ props.title }}</Heading>
-      <div :class="$style['updatedAt']">{{ props.updatedAt }}</div>
+  <div :class="$style.note">
+    <div :class="$style.cover"></div>
+    <div :class="$style.footer">
+      <Heading
```

Comment:
Please, don't use `<Heading>` here. Use regular `<div>` since it is not a Heading in Figma.

You can find text styles at the right-side panel in Figma. For example, here is how Heading style looks:

![image](https://github.com/codex-team/notes.web/assets/3684889/e952f987-f210-49e7-a589-d13c1645ea45)


And in Card there is just ui-base-bold:

![image](https://github.com/codex-team/notes.web/assets/3684889/6d6f88e3-83bb-426f-a551-d4b3e1ac0bac)